helm/ako/crds/ako.vmware.com_l4rules.yaml

```diff
@@ -0,0 +1,245 @@
+                        each Service Engine and mark a server up or down based on
+                        the response. The Passive monitor listens only to client to
+                        server communication. It raises or lowers the ratio of traffic
+                        destined to a server based on successful responses.
+                      items:
+                        type: string
+                      type: array
+                    inlineHealthMonitor:
+                      default: true
+                      description: The Passive monitor will monitor client to server
+                        connections and requests and adjust traffic load to servers
+                        based on successful responses.  This may alter the expected
+                        behavior of the LB method, such as Round Robin.
+                      type: boolean
+                    lbAlgorithm:
+                      default: LB_ALGORITHM_LEAST_CONNECTIONS
+                      description: The load balancing algorithm will pick a server
+                        within the pool's list of available servers. Values LB_ALGORITHM_NEAREST_SERVER
+                        and LB_ALGORITHM_TOPOLOGY are only allowed for GSLB pool
+                      enum:
+                      - LB_ALGORITHM_LEAST_CONNECTIONS
+                      - LB_ALGORITHM_ROUND_ROBIN
+                      - LB_ALGORITHM_FASTEST_RESPONSE
+                      - LB_ALGORITHM_CONSISTENT_HASH
+                      - LB_ALGORITHM_LEAST_LOAD
+                      - LB_ALGORITHM_FEWEST_SERVERS
+                      - LB_ALGORITHM_RANDOM
+                      - LB_ALGORITHM_FEWEST_TASKS
+                      - LB_ALGORITHM_NEAREST_SERVER
+                      - LB_ALGORITHM_CORE_AFFINITY
+                      - LB_ALGORITHM_TOPOLOGY
+                      type: string
+                    lbAlgorithmHash:
+                      description: Criteria used as a key for determining the hash
+                        between the client and  server.
+                      enum:
+                      - LB_ALGORITHM_CONSISTENT_HASH_SOURCE_IP_ADDRESS
+                      - LB_ALGORITHM_CONSISTENT_HASH_SOURCE_IP_ADDRESS_AND_PORT
+                      - LB_ALGORITHM_CONSISTENT_HASH_URI
+                      - LB_ALGORITHM_CONSISTENT_HASH_CUSTOM_HEADER
+                      - LB_ALGORITHM_CONSISTENT_HASH_CUSTOM_STRING
+                      - LB_ALGORITHM_CONSISTENT_HASH_CALLID
+                      type: string
+                    minServersUp:
+                      description: Minimum number of servers in UP state for marking
+                        the pool UP.
+                      type: integer
+                    pkiProfileRef:
+                      description: Avi will validate the SSL certificate present by
+                        a server against the selected PKI Profile.
+                      type: string
+                    port:
```

Comment:
@swathinsankaran: What is `port` and `protocol` field signifies at Avi controller pool side? Is these are properties of pool-server or something else?

Reply:
It signifies the backend port and protocol as the port and protocol come under backendProperties.

Reply:
Sorry. I didn't get that. when you say `backendPort`, is it related with pool server populated?